docs/content/node-operation/node-bootstrap.mdx

```diff
@@ -262,7 +279,7 @@ This is the recommended way to start your node for the first time.
 3. Configure Dynamic Startup by adding flags:
 ```shell:title=ExampleDynamicStartupFlags
   ... \
-  --dynamic-startup-access-address=access.flow.com:9001 \
```

Comment:
`access.flow.com` doesn't exist today. So, I replaced it with access-001. Plus, the key is specific to that node.

Reply:
The public key I have here is not a real value either, it's just a placeholder/example. We would need to change that as well to make this a usable example. 

If we use the `mainnet16` address, we will need to update it every spork. I remember we talked about adding a `secure.access...` address for use in the DKG, which would be:
* the same address sporks
* point to the same node across sporks (eg. AN1)

Do you remember if we ever got that set up?

Reply:
yup - thanks for the reminder. There is indeed a secure hostname that points to AN7 (`secure.mainnet.nodes.onflow.org`). I will update the doc.

Reply:
Here is the network key for AN7: `28a0d9edd0de3f15866dfe4aea1560c4504fe313fc6ca3f63a63e4f98d0e295144692a58ebe7f7894349198613f65b2d960abf99ec2625e247b1c78ba5bf2eae`

For reference, here is a script to retrieve the network key from the protocol state snapshot in the right format:
```
cat ./root-protocol-state-snapshot.json | jq '.Identities[] | select(.Address == "access-007.mainnet16.nodes.onflow.org:3569") | .NetworkPubKey' | sed 's/\"//g' | base64 --decode | od -t x1 -An | sed 's/ //g' | tr -d '\n'
```